storage/mongo/index.js

```diff
@@ -1 +1,7 @@
-// JavaScript source code
+var MongoUtil = require('utils.js');
+
+module.exports = function (config) {
+    init = function (config) {
```

Comment:
Same thing goes here
<pre>
init: function(){
return when.promise(function(reject,resolve){
//Here as mongoutils connect to server is promise you wuill get to know using .then function and .otherwise function whethr it was resolved or rejected. see when.js library
})
}
</pre>